src/qibo/hamiltonians.py

```diff
@@ -69,9 +36,9 @@ def XXZ(nqubits, delta=0.5, numpy=False, trotter=False):
             h = XXZ(3) # initialized XXZ model with 3 qubits
     """
     if trotter:
-        hx = np.kron(matrices.X, matrices.X)
-        hy = np.kron(matrices.Y, matrices.Y)
-        hz = np.kron(matrices.Z, matrices.Z)
+        hx = qnp.kron(matrices.X, matrices.X)
+        hy = qnp.kron(matrices.Y, matrices.Y)
+        hz = qnp.kron(matrices.Z, matrices.Z)
```

Comment:
For example here, if we implement jax we may prefer to use K.kron in order to use the GPU allocation.

Reply:
This is just a comment, from a practical perspective we don't need this level of flexibility for these objects.